src/servers/src/prom_row_builder.rs

```diff
@@ -85,7 +85,7 @@ impl Default for TableBuilder {
 
 impl TableBuilder {
     pub(crate) fn with_capacity(cols: usize, rows: usize) -> Self {
-        let mut col_indexes = HashMap::with_capacity(cols);
+        let mut col_indexes = HashMap::with_capacity_and_hasher(cols, Default::default());
```

Comment:
I am wondering if we could improve the performance of our memtable implementations by replacing the current `HashMap`.

Reply:
> I am wondering if we could improve the performance of our memtable implementations by replacing the current `HashMap`.

hashbrown and fxhashmap are definitely worth trying when the key is not arbitrary user input data.